src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return (days * 40) - 50;
+  } else if (days > 2) {
+    return (days * 40) - 20;
+  } else {
+    return days * 40;
+  }
```

Comment:
this does not make much sense, if you return the value, then you do not really need `else if` or `else`

Reply:
also let's not have magic numbers in the code. at the moment i dont understand what 20, 40 and 50 are